app/code/Magento/LoginAsCustomerGraphQl/Model/Resolver/RequestCustomerToken.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\LoginAsCustomerGraphQl\Model\Resolver;
+
+use Magento\Framework\AuthorizationInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\LoginAsCustomerApi\Api\ConfigInterface;
```

Comment:
I would recommend using aliases for such imports. Just to make it more clear what config interface your code is about. 
```suggestion
use Magento\LoginAsCustomerApi\Api\ConfigInterface as LoginAsCustomerConfig;
```